projects/1.20/assets/food-expansion-reimagined/foodexpansion/lang/zh_cn.json

```
@@ -0,0 +1,47 @@
+{
+    "item.foodexpansion.jelly": "翡翠果冻",
+    "item.foodexpansion.bacon": "培根",
+    "item.foodexpansion.cooked_bacon": "脆皮培根",
```

```suggestion
    "item.foodexpansion.cooked_bacon": "香脆培根",
```
一般来说描述`烤的酥脆的培根`不用`脆皮`，培根是不带皮的
`脆皮五花肉`则是真的把皮做脆了